test/e2e/README.md

```diff
@@ -192,6 +192,35 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on existing K8s cluster
+
+If you already have a K8s cluster and push antrea on it, the following steps should be done for running the e2e tests or other kinds of tests.
```

Comment:
`push antea on it` -> want to push antea on it?

Reply:
The condition should be the antrea has been pushed on the cluster, maybe the statement is not clear.